README.md

```diff
@@ -69,10 +69,10 @@ https://docs.python.org/3/library/argparse.html#formatter-class):
 The default styles used by *rich-argparse* are carefully chosen to work in different light and dark
 themes.
 
-### Customize the colors
+### Customize the font style and colors
 
-You can customize the colors in the output by modifying the `styles` dictionary on the formatter
-class. *rich-argparse* defines the following styles:
+You can customize the font style and colors of the output by modifying the values of the `styles` dictionary on the formatter
+class:
```

Comment:
Same here, "font style" is not correct.

You also removed the sentence that explains what the next example is about: "*rich-argparse* defines the following styles:". This should not be removed.

I suggest using more accurate wording like this:

```md
You can customize the styles of the output by modifying the `styles` dictionary on the formatter
class. You can use any rich style as defined [here](https://rich.readthedocs.io/en/latest/style.html).
*rich-argparse* defines the following styles:
```

The rich styles page has a link to the list of colors at the top, so no need to link to the colors page below.